samples/d3/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "d3",
+  "type": "widget",
+  "version": "1.0.0",
```

Comment:
All widgets use the same version make it easy to manage

Reply:
Thanks, I updated all the `version` properties to be `1.1.0`.